### DISCUSSION TITLE
[T2] Duda Parte 1 largo strings cortos

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#132] [Tarea 2] [answered]
Buenas tardes, tengo una duda respecto al largo que van a tener los strings más pequeños, van a tener un largo máximo? O la relación entre el largo máximo y cantidad de strings pequeños qué se van a buscar. 
Esto es para saber que tan grande debiese ser la tabla de hash, para evitar choques o como manejar estos para que la tabla no quede muy vacía.
Agradezco cualquier información respecto a esto.

**Comment by sebacarrascop** [comment, marked as answer]
hola, solo se sabe que el largo M es menor al largo N, a partir de eso debes buscar una estrategia para tu función de hash. Ten en cuenta que todas las consultas serán del largo M.

**Reply by MaxAl100** [reply]
Ok, muchas gracias 😅
